e2e-tests/cypress/e2e/journeys/insurance/eligibility/eligible-to-apply-online.spec.js

```diff
@@ -3,24 +3,46 @@ import { insurance } from '../../../pages';
 import partials from '../../../partials';
 import { PAGES } from '../../../../../content-strings';
 import { ROUTES } from '../../../../../constants';
+import { completeAndSubmitBuyerCountryForm } from '../../../../support/forms';
+import {
```

Comment:
is there a reason these are imported and not cy. commands?

Reply:
All of these "complete X form" functions are from insurance eligibility, built ages ago, they are called in `submitInsuranceEligibilityAnswersHappyPath` cypress command which is the one that's used the most.

Currently all the individual functions (listed here), are not cypress commands, but they could be.

We should update all insurance eligibility E2E tests to use cypress commands instead of importing the functions like this, but not a quick thing to do.

Can we do this later so that we can get this PR merged and unblock the pipeline? Can create a ticket.

Reply:
As discussed created a future ticket for this https://ukef-dtfs.atlassian.net/browse/EMS-1156